.github/workflows/image.yml

```diff
@@ -0,0 +1,43 @@
+on:
+  pull_request:
+  push:
+    branches: [master]
+jobs:
+  build:
+    strategy:
+      matrix:
+        # the arm64 build takes too long, so disable for now
+        arch: [amd64]
+    runs-on: ubuntu-latest
+    env:
+      IMG_CACHE: ghcr.io/getsentry/relay:${{ matrix.arch }}-latest
+      IMG_VERSIONED: ghcr.io/getsentry/relay:${{ matrix.arch }}-${{ github.sha }}
+    steps:
+    - uses: actions/checkout@v3
+    - run: docker run --rm --privileged tonistiigi/binfmt --install arm64
```

Comment:
Optional: Should this be removed for now as this is just amd64?

Reply:
I figure it's simpler to leave it, and if we want arm64 we can just adjust the matrix